neural_lam/train_model.py

```diff
@@ -201,15 +191,26 @@ def main(input_args=None):
         help="""JSON string with variable-IDs and lead times to log watched
              metrics (e.g. '{"1": [1, 2], "3": [3, 4]}')""",
     )
+    parser.add_argument(
+        "--include_past_forcing",
+        type=int,
+        default=1,
+        help="Number of past time steps to use as input for forcing data",
+    )
+    parser.add_argument(
+        "--include_future_forcing",
+        type=int,
+        default=1,
+        help="Number of future time steps to use as input for forcing data",
+    )
     args = parser.parse_args(input_args)
     args.var_leads_metrics_watch = {
         int(k): v for k, v in json.loads(args.var_leads_metrics_watch).items()
     }
-    config_loader = config.Config.from_file(args.data_config)
 
     # Asserts for arguments
+    assert args.config is not None, "Specify your config with --config_path"
```

Comment:
Should this be `args.config_path` instead. The `args` Namespace doesn't have a `config` attribute currently.